### ISSUE TITLE
MPII 90.3 in the paper cannot be reproduced

### ISSUE THREAD

**Issue by ll8657** [leoxiaobin/deep-high-resolution-net.pytorch#306]
The MPII dataset training can only achieve the accuracy of 89.8 on MPII VAL, which cannot reach the 90.3 in the paper
